core/domain/question_services.py

```diff
@@ -0,0 +1,205 @@
+# coding: utf-8
+#
+# Copyright 2017 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Services for questions data model."""
+
+import logging
+
+from core.domain import question_domain
+from core.platform import models
+
+import feconf
+
+(question_models,) = models.Registry.import_models([models.NAMES.question])
+
+# This takes additional 'title' parameters.
+CMD_CREATE_NEW = 'create_new'
+
+
+def _create_question(committer_id, question, commit_message):
+    """Creates a new question.
+
+    Args:
+        committer_id: str. ID of the committer.
+        question: Question. question domain object.
+        commit_message: str. A description of changes made to the question.
+    TODO: implement commit_cmds.
+    """
+    model = question_models.QuestionModel.create(
+        title=question.title,
+        question_data=question.question_data,
+        question_data_schema_version=question.question_data_schema_version,
+        collection_id=question.collection_id,
+        language_code=question.language_code,
+    )
+
+    model.commit(committer_id, commit_message, [{
+        'cmd': CMD_CREATE_NEW,
+        'title': question.title
+        }])
+    return model
+
+
+def add_question(committer_id, question):
+    """Saves a new question.
+    Args:
```

Comment:
Add blank line above "Args". Ditto below. Please follow existing formats, it reduces review back-and-forth.

Reply:
Updated. My bad.